holesky/.env.example

```diff
@@ -1,4 +1,4 @@
-MAIN_SERVICE_IMAGE=ghcr.io/layr-labs/eigenda/opr-node:0.8.4
```

Comment:
why isn't this already on `0.8.5-rc.0`?

Reply:
I merged to master when doing the testnet release, and create a rc branch from the master. 

Do you want to do branch then master instead this time. I am fine with that